content/stuff/wordpress-widget/contents.lr

```diff
@@ -12,7 +12,7 @@ space: Coredump
 ---
 space_url: https://www.coredump.ch/
 ---
-category: integration
+category: space-visualization, operation
```

Comment:
Yes it is, for me this is also for project that gives you (as in you as a space) the possibility to communicate with the public. I think about this category as in "everything that helps you provide an endpoint and gets data in and data out for public communication". 
Like not the hackerspaces app (which also is kinda public communication) but something you as a space provide and use to communicate with the public (e.g. a twitter bot that tweets when your space opens). Something that makes "operation" easier (so far i didn't find a better word for it, operation is not it but i didn't find a better one)

Reply:
I still think it feels off, since everything else on the page explains how to get your space listed in the directory, while this is the only example that *consumes* the endpoint instead of feeding it.

Reply:
I would still like to get more projects like this and feature them!

The more we can help spaces make the data easily accessible, the less work they have. 
I just don't know how to put that down right now

Reply:
Yes, but still, showing your opening status on the website is definitely "using the SpaceAPI" and not "providing an endpoint to the SpaceAPI".

The "how to use" page is already all about getting data out of the API for public communication, and the "provide an endpoint" page is all about getting your entry *into* the API. I don't see why we should start mixing those two.